### DISCUSSION TITLE
Can we make the sidebar arrow in theme-simple point right if it has multiple sidebar .md files

### DISCUSSION THREAD

**Discussion by SumithChandran** [docsifyjs/docsify#1862] [General]
Is it possible to make the sidebar arrow in `theme-simple` link to point right if it has multiple sidebar .md files. The arrow is pointing down during my testing.
![Sidebar arrow](https://user-images.githubusercontent.com/46446418/185608611-8bb6099e-538e-42b5-b685-290a95ea80ca.jpg)
![image](https://user-images.githubusercontent.com/46446418/185608689-f46420f8-0131-4ac7-af65-e70cfbcd7aea.png)

Here are the files I used for testing.
[sidebar test.zip](https://github.com/docsifyjs/docsify/files/9381753/sidebar.test.zip)

**Comment by trusktr** [comment]
Seems funky to show something open when it has nothing perhaps. What would be a good solution? Gray out the triangle maybe?

**Comment by SumithChandran** [comment]
@trusktr thanks for looking on this.
When ever I load the page, the arrow is **pointing downward** (`file1`).
![image](https://user-images.githubusercontent.com/46446418/185901671-79508bdf-5126-4280-8a69-d92707983241.png)

I want this arrow to **point right**  like this:
![Screenshot 2022-08-22 160934](https://user-images.githubusercontent.com/46446418/185902667-80ae8e65-54d5-415e-92b6-9c02ef989b24.png)
when I load the page for the 1st time.

Any ideas or thoughts.
